### PR TITLE
feat: adjust the sort label color based on the header background color

### PR DESCRIPTION
**Pull request by LiKang6688**
When you set the color of header content, the color of the sort label should be the same as that.
When there is no header content color setting, the sort label color is kind of white in the dark background color and black in the light background color.

### CODE REVIEW

File: src/table/components/TableHeadWrapper.jsx [hunk 1]
```diff
@@ -40,7 +40,15 @@ function TableHeadWrapper({
...
-
+  // When you set the header font color,
+  // the sort label color should be same.
+  // When there is no header content color setting,
+  // the sort label color is depending on the header background color.
+  const tableSortLabelStyle = {
+    '&.Mui-active .MuiTableSortLabel-icon': {
+      color: headerStyle.color ? headerStyle.color : headerStyle.sortLabelColor,
+    },
+  };
```
Comment (haxxmaxx) [inline src/table/components/TableHeadWrapper.jsx:51]: Move to styling-utils

File: src/table/components/TableHeadWrapper.jsx [hunk 2]
```diff
@@ -40,7 +40,15 @@ function TableHeadWrapper({
...
   };
   const headerStyle = useMemo(() => getHeaderStyle(layout, theme), [layout, theme.name()]);
-
+  // When you set the header font color,
+  // the sort label color should be same.
+  // When there is no header content color setting,
+  // the sort label color is depending on the header background color.
+  const tableSortLabelStyle = {
+    '&.Mui-active .MuiTableSortLabel-icon': {
+      color: headerStyle.color ? headerStyle.color : headerStyle.sortLabelColor,
```
Comment (haxxmaxx) [inline src/table/components/TableHeadWrapper.jsx:49]:
```suggestion
      color: headerStyle.color ?? headerStyle.sortLabelColor,
```

File: src/table/components/TableHeadWrapper.jsx (hunk 1 above)
Comment (LiKang6688) [reply src/table/components/TableHeadWrapper.jsx]: Done

File: src/table/components/TableHeadWrapper.jsx [hunk 3]
```diff
@@ -40,6 +40,11 @@ function TableHeadWrapper({
     },
   };
   const headerStyle = useMemo(() => getHeaderStyle(layout, theme), [layout, theme.name()]);
+  const tableSortLabelStyle = {
+    '&.Mui-active .MuiTableSortLabel-icon': {
+      color: headerStyle.sortLabelColor,
+    },
+  };
```
Comment (haxxmaxx) [inline src/table/components/TableHeadWrapper.jsx:47]: This is fine, but I have a refactor idea where we move all classes made for sx to styling-utils. So you get everything already packaged, and only updated when needed

**Review by haxxmaxx — APPROVED** [review]

**Pull request merged**